tooling/reproducible/macos_repro_build_compare.sh

```diff
@@ -0,0 +1,666 @@
+      target_file="$(echo -e "${fixed_value}" | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
+      fixed_value=" $fixed_value"
+    fi
+    if [ "$fixed_param" == "--freetype-dir" ]; then fixed_value=" $fixed_value" ; fi
+    if [ "$fixed_param" == "--user-openjdk-build-root-directory" ]; then fixed_value=" $WORK_DIR/temurin-build/workspace/build/openjdkbuild/ " ; fi
+    if [ "$fixed_param" == "--build-variant" ]; then fixed_value=" $fixed_value " ; fi
+    if [ "$fixed_param" == "--build-reproducible-date" ]; then fixed_value=" $fixed_value" ; fi
+
+    if containsElement "$fixed_param" "${CONFIG_ARGS[@]}"; then
+      # Check if fixed_param is in CONFIG_ARGS
+      STRINGTOADD="$fixed_param$fixed_value"
+      CONFIG_ARRAY+=("$STRINGTOADD")
+    elif containsElement "$fixed_param" "${NOTUSE_ARGS[@]}"; then
+      # Check if fixed_param is in NOTUSE_ARGS
+      STRINGTOADD="$fixed_param$fixed_value"
+      IGNORED_ARRAY+=("$STRINGTOADD")
+    elif containsElement "$fixed_param" "${FINAL_ARG[@]}"; then
+      # Check if parameter should be at the end, usually just version & variant
+      STRINGTOADD="$fixed_param$fixed_value"
+      FINAL_ARRAY+=("$STRINGTOADD")
+    else
+      # Neither Config Or Ignore, so Build
+      STRINGTOADD="$fixed_param$fixed_value"
+      BUILD_ARRAY+=("$STRINGTOADD")
+    fi
+  done
+
+  # Construct Final Parameter String
+  for element in "${BUILD_ARRAY[@]}"; do
+    build_string+="$element"
+  done
+
+  for element in "${CONFIG_ARRAY[@]}"; do
+    config_string+="$element"
+  done
+
+  for element in "${FINAL_ARRAY[@]}"; do
+    final_string+="$element"
+  done
+
+  final_params="$build_string --configure-args \"$config_string\" $final_string"
+
+echo ""
+  echo "Make JDK Any Platform Argument List = "
+  echo "$final_params"
+  echo ""
+  echo "Parameters Parsed Successfully"
+}
+
+Build_JDK() {
+  echo "Building JDK..."
+
+  # Trigger Build
+  cd "$WORK_DIR"
+  echo "cd temurin-build && ./makejdk-any-platform.sh $final_params 2>&1 | tee build.$$.log" | sh
+  # Copy The Built JDK To The Working Directory
+  cp $WORK_DIR/temurin-build/workspace/target/"$target_file" $WORK_DIR/reproJDK.tar.gz
+}
+
+Compare_JDK() {
+  echo "Comparing JDKs"
+  echo ""
+  mkdir "$WORK_DIR/compare"
+  cp $WORK_DIR/src_jdk_dist.tar.gz $WORK_DIR/compare
+  cp $WORK_DIR/reproJDK.tar.gz $WORK_DIR/compare
+  
+
+  # Get The Current Versions Of The Reproducible Build Scripts
+  wget -O "$WORK_DIR/compare/repro_common.sh" "https://raw.githubusercontent.com/adoptium/temurin-build/master/tooling/reproducible/repro_common.sh"
+  wget -O "$WORK_DIR/compare/repro_compare.sh" "https://raw.githubusercontent.com/adoptium/temurin-build/master/tooling/reproducible/repro_compare.sh"
+  wget -O "$WORK_DIR/compare/repro_process.sh" "https://raw.githubusercontent.com/adoptium/temurin-build/master/tooling/reproducible/repro_process.sh"
+
+  # Set Permissions
+  chmod +x "$WORK_DIR/compare/"*sh
+  cd "$WORK_DIR/compare"
```

Comment:
Those files are part of  tool/reproducible. No need to require. Especially it's hard code to temurin-build/master, which won't take developer's PRs.

Reply:
I'll update the script, so it checks for their existence locally, and only download the versions from git, if they are not found for any reason :)

Reply:
This change is in the current version of the script.